src/views/Simulation/SimulationViewHook.js

```diff
@@ -93,6 +93,7 @@ export const useSimulationView = () => {
     settings.outputLevels,
     settings.orientation,
     settings.spacing,
+    settings.filterEffectActivated,
```

Comment:
The property name can be confusing, because of the other setting named `graphViewFilters`. Maybe we can rename `filterEffectActivated` to `enableGlowEffect`?